src/components/projecteditor/control.js

```diff
@@ -696,10 +696,12 @@ export default class DevkitProjectEditorControl extends Component {
         projectItem.save((status)=>{
             if(status==0) {
                 // TODO: this lookup is bad.
-                const ctrs=projectItem.props.state.children[0].props.state._children;
-                const contract=ctrs[ctrs.length-2];
-                const item=contract.props.state.children[0];
-                if(this.props.router.panes) this.props.router.panes.openItem(item);
+                const ctrs=projectItem.props.state.children[1].props.state._children;
+                if(ctrs && ctrs.length >= 2) {
```

Comment:
@filippsen could you add the comments here of why choosing .children[1] and ctrs.lenghth >=2 as it nos not obvious in any sense for the reader of this code? With a small comment in the code there wont be any more questions of why those values.